test/remote/account_preverified_token_tests.js

```diff
@@ -2,12 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-var test = require('../ptaptest')
+var test = require('tap').test
 var TestServer = require('../test_server')
 var Client = require('../client')
 var JWTool = require('fxa-jwtool')
 
-process.env.TRUSTED_JKUS = 'http://127.0.0.1:9000/.well-known/public-keys'
```

Comment:
@rfk @vbudhram this is another case where preVerification or verification fails. I tried moving this trusted JKU to config but then I got an `unknown kid` error. ideas?